percona-packaging/redhat/percona-server-mongodb.spec.template

```diff
@@ -394,10 +397,24 @@ fi
 %endif
     fi
 fi
+mkdir -p %{psmdb_telemetry}
+chown mongod:percona-telemetry /usr/local/percona/telemetry/psmdb
+chmod 775 /usr/local/percona/telemetry/psmdb
+chmod g+s /usr/local/percona/telemetry/psmdb
+chmod u+s /usr/local/percona/telemetry/psmdb
 cp %SOURCE999 /tmp/ 2>/dev/null || :
 bash /tmp/call-home.sh -f "PRODUCT_FAMILY_PSMDB" -v "@@VERSION@@-@@RELEASE@@" -d "PACKAGE" &>/dev/null || :
+chgrp percona-telemetry /usr/local/percona/telemetry_uuid
+chmod 664 /usr/local/percona/telemetry_uuid
 rm -f /tmp/call-home.sh
 
+%post mongos
+mkdir -p /usr/local/percona/telemetry/psmdbs
+chown mongod:percona-telemetry /usr/local/percona/telemetry/psmdbs
+chmod 775 /usr/local/percona/telemetry/psmdbs
+chmod g+s /usr/local/percona/telemetry/psmdbs
+chmod u+s /usr/local/percona/telemetry/psmdbs
```

Comment:
install -d -m 2775 -o mongod -g percona-telemetry /usr/local/percona/telemetry/psmdbs

Reply:
Good point, Thank you.